src/main/java/edu/arizona/kra/subaward/batch/service/impl/GlDataImportServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package edu.arizona.kra.subaward.batch.service.impl;
+
+import edu.arizona.kra.subaward.batch.InvoiceFeedUtils;
+import edu.arizona.kra.subaward.batch.bo.BiGlEntry;
+import edu.arizona.kra.subaward.batch.bo.UAGlEntry;
+import edu.arizona.kra.subaward.batch.dao.BiGLFeedDao;
+import edu.arizona.kra.subaward.batch.dao.SubawardGlFeedDao;
+import edu.arizona.kra.subaward.batch.service.GlDataImportService;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.kuali.rice.krad.service.BusinessObjectService;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.sql.Date;
+import java.util.List;
+
+/**
+ * Created by nataliac on 8/8/18.
+ */
+public class GlDataImportServiceImpl implements GlDataImportService {
+    private static final Log LOG = LogFactory.getLog(GlDataImportServiceImpl.class);
+
+    private BiGLFeedDao biGLFeedDao;
+    private SubawardGlFeedDao subawardGLFeedDao;
+
+    private BusinessObjectService businessObjectService;
+
```

Comment:
We will have a couple of additional steps for those purposes... was still working with Tom to identify all types of errors that need to be reported. With the next step planning to implement an ErrorReporting service to output all erros in the corresponding table

Reply:
OK, but the description for UAR-2691 mentions updating the temp table and the error table, so you might want to change that. It would make sense to pare this Jira and PR down to just what is involved in importing the GL data.

Reply:
I've updated the description accordingly.